docs/dependency_confusion.md

```diff
@@ -1,140 +1,55 @@
+during software builds.
 
-Hopefully, you are building your application in an isolated environment (such as a container build
-with no network access) to protect your own system. In the best case scenario, the malicious behaviour
-will manifest at build time and the build will fail. In the worst case scenario, the build will succeed
-but the built product will contain malicious code, which may then affect whoever tries to run it.
+## Dependency Confusion and cachi2
 
-## Dependency Confusion vs Cachi2
+_cachi2 is as vulnerable to dependency confusion as your package manager, not more, not less._
 
-Before delving into the specifics of each supported package manager, let us make a general statement
-about Cachi2’s stance towards dependency confusion.
+cachi2 downloads software dependencies, doing its best to prevent any arbitrary code execution
+that can be part of the dependency itself and could potentially infect the resulting product,
+for example, by running a pre/post-install script, which requires detailed research on whether
+a package manager can be fully trusted and relied upon to download dependencies using its native
+mechanisms or by a workaround implemented as an abstraction layer.
 
-Cachi2 aims to match the behaviour of your package manager as closely as possible. Generally
-speaking, Cachi2 is as vulnerable to dependency confusion as your package manager, not more, not
-less. However, all the supported package managers provide means to protect yourself by following
-best practices. For example, you can remove all known attack vectors by simply verifying the
-checksums of your dependencies.
+The impact on the build depends on the built environment. Ideally, the application should be built
+in an isolated environment, such as a container without network access. In this case, if malicious
+behavior is present, it may cause the build to fail, protecting your system from further harm.
+If the build succeeds, the resulting product could contain malicious code, posing a risk to anyone
+who runs it.
 
-Cachi2 enforces *some* of these best practices. Typically, each package manager provides a lockfile
-or other mechanism for pinning dependency versions. Cachi2 requires this. Verifying checksums is
-usually up to you.
+## Package Managers
 
-# Package Managers
+cachi2 follows some of the best practices, such as verifying the checksums of your dependencies
+to ensure their integrity or forcing you to pin all your dependencies to exact versions to avoid
+unexpected updates. However, it is essential to understand how a package manager works and
+what security features it provides to protect your software supply chain. cachi2 supports the
+following package managers:
 
-## gomod
+### gomod
 
-*TL;DR: commit your go.sum.*
+- package manager for Go
+- lock file: `go.sum`
+- [docs/gomod.md](https://github.com/containerbuildsystem/cachi2/tree/main/docs/gomod.md)
```

Comment:
If you remove the whole dependency-related content from the respective package manager sections, what is the actual value of having these like you propose at all?